package.json

```diff
@@ -57,10 +61,10 @@
     "inquirer": "^7.0.0",
     "istanbul": "^0.4.5",
     "node-fetch": "^2.3.0",
-    "openzeppelin-solidity": "2.3.0",
+    "openzeppelin-solidity": "3.0.1",
```

Comment:
Though both `openzeppelin-solidity` and `@openzeppelin/contracts` npm repositories are still being updated, `@openzeppelin/contracts` seems too be the preferred repo now (it's the one mentioned on their Github page).

Reply:
I had no idea about the new one. Will update.